packages/web/docs/src/pages/product-updates.mdx

```diff
@@ -0,0 +1,24 @@
+import { useData } from '@theguild/components'
+import { ProductUpdateTeaser } from '../components/product-updates'
+
+export { getStaticProps } from '../components/product-updates'
+
+export function ProductUpdatesPage() {
+  const { changelogs } = useData()
+  return (
+    <ol className="relative mt-12 border-l border-gray-200 dark:border-gray-700">
+      {changelogs.map(item => (
+        <ProductUpdateTeaser key={item.route} {...item} />
+      ))}
+    </ol>
+  )
+}
+
+<div className="[&>h1]:!text-left">
+  # Product Updates
+
+The most recent developments from GraphQL Hive.
+
+</div>
```

Comment:
changed little bit styles, to use Nextra's typesetting styles
## before
<img width="1617" alt="image" src="https://github.com/user-attachments/assets/4cb755cb-2d59-49b9-836c-962c18757ffa">

## now

<img width="1594" alt="image" src="https://github.com/user-attachments/assets/2f15a6f3-bb23-45a6-a3fb-58be58b14a37">

Reply:
The spacing between the heading and the first paragraph looks worse now IMHO.